draftfast/showdown/orm.py

```diff
@@ -13,8 +13,6 @@ def __init__(self, player: Player, captain: bool = False):
             self.real_pos = self.pos
             self.pos = 'CPT'
             self.captain = True
-            self.proj *= 1.5
```

Comment:
I think this is a change we want. I question hard-coding a 1.5 - the drawback I guess is that the rules of DraftKings are no longer "exposed" anywhere - but if we are going to expose them, it should be done as something you can pass into the initializer or maybe in `RuleSet`.

Reply:
I thought per #140 the CSVs provided by DK include CPT entires?

Reply:
They do - I think I want to keep `ShowdownPlayer` though since `real_pos` is valuable and used by the optimizer.